src/Controller/Action/AddSelectedProductsToCartAction.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+* This file was created by developers working at BitBag
+* Do you need more information about us and what we do? Visit our https://bitbag.io website!
+* We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Controller\Action;
+
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\AddSelectedProductsToCart;
+use BitBag\SyliusWishlistPlugin\Context\WishlistContextInterface;
+use BitBag\SyliusWishlistPlugin\Form\Type\WishlistCollectionType;
+use BitBag\SyliusWishlistPlugin\Processor\WishlistCommandProcessorInterface;
+use Sylius\Component\Order\Context\CartContextInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+final class AddSelectedProductsToCartAction
+{
+    private WishlistContextInterface $wishlistContext;
+
+    private CartContextInterface $cartContext;
+
+    private FormFactoryInterface $formFactory;
+
+    private FlashBagInterface $flashBag;
+
+    private MessageBusInterface $commandBus;
+
+    private WishlistCommandProcessorInterface $wishlistCommandProcessor;
+
+    private UrlGeneratorInterface $urlGenerator;
+
+    public function __construct(
```

Comment:
This class is almost identical to `AddProductsToWishlistAction`: refactor this to design the same logic only once. We don't like copy-paste code.